js/background.js

```diff
@@ -98,6 +99,10 @@ function checkRedirects(details) {
 			}
 			ignoreNextRequest[result.redirectTo] = new Date().getTime();
 			
+			if (result.redirectTo.match(internalProtocolsRegexp) == null) {
+				window.setTimeout(function(){chrome.tabs.remove(details.tabId);}, 5000);
```

Comment:
The timeout is for the case when the external application opens for a first time and awaits for the confirmation from user.